.pre-commit-config.yaml

```diff
@@ -7,7 +7,7 @@ repos:
     - id: no-commit-to-branch
       args: ['--branch', 'main']
     - id: check-added-large-files
-      args: ['--maxkb=2000']
+      args: ['--maxkb=20000']
```

Comment:
Why is it necessary? Are you committing any especially large files?

Reply:
yes, rewardbench inputs are >2MB , so I changed to 20